examples/playwright/test/arith.test.js

```diff
@@ -18,7 +18,7 @@ test.describe.parallel('Test suite. Level 1', () => {
     expect(mul(3, 4)).toBe(12);
   });
 
-  test('3 * 4 = 12', async () => {
+  test('3 * 5 = 15', async () => {
```

Comment:
doesn't match the expression in the test

Reply:
fixed